docs/options.md

```diff
@@ -156,7 +156,7 @@ or
 Default | CLI Override | API Override
 --------|--------------|-------------
 `false` | `--require-pragma` | `requirePragma: <bool>`
-<!-->
```

Comment:
Actually this section is supposed to be hidden because the feature is unreleased. I think you just need to delete that `>` instead.

Reply:
Ah, I see! I was already wondering why Prettier is telling me: "Ignored unknown option: insertPragma".